### PR TITLE
 Multiple Window User Guide 

### PR DESCRIPTION
**Pull request by hrdisaac**

### CODE REVIEW

**Review by adeelkazmi — APPROVED** [review]

File: docs/nui-tutorials/window.md [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+The size of a window can be retrieved, and the window can be resized.
+
+~~~{.cs}
+// Retrieve the current size of the window
+Size2D windowSize = window.WindowSize;
+
+// Resize the window by increasing its width
+windowSize.Width += 100;
+window.WindowSize = windowSize;
+~~~
```
Comment (agnelovaz) [inline docs/nui-tutorials/window.md:145]: Add a link to the ResizeEvent chapter.

File: docs/nui-tutorials/window.md [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+    Title = "new window"
+};
+~~~
+
+[Back to top](#top)
+
+<a name="deleting"></a>
+## Deleting window
+
+The main window cannot be deleted, otherwise it will cause unexpected behaviour.
```
Comment (agnelovaz) [inline docs/nui-tutorials/window.md]: Update comment as not unexpected behavior (update TizenFX API comment too)

File: docs/nui-tutorials/window.md [hunk 3]
```diff
@@ -0,0 +1,388 @@
...
+<a name="focusmanagement"></a>
+## Focus management
+
+Keyboard focus can be handled between multiple windows.
+
+Here is a simple example to show how this can be done.
+
+<img src="{{site.baseurl}}/assets/images/window/FocusManagement.png">
+
+Suppose there are two windows and each window has two buttons (as shown in the above diagram). We want to move focus between the buttons in these two windows using "Left" and "Right" navigation keys.
```
Comment (agnelovaz) [inline docs/nui-tutorials/window.md]: Move this to above the image and update (as shown in the ..... diagram)

File: docs/nui-tutorials/window.md [hunk 4]
```diff
@@ -0,0 +1,388 @@
...
+<a name="mainwindow"></a>
+## Creating main window
+
+When an application is created, the main window will be automatically created:
+
+~~~{.cs}
+MyApplication myApp = new MyApplication();
+~~~
+
+By default, the main window is created in full screen size, except that if running on the desktop the default size of the main window is set by the **set-env** script. The default main window size can be overriden by environment variables or command line options when launching the application in command line. For example:
```
Comment (agnelovaz) [inline docs/nui-tutorials/window.md:47]: "By default, the main window is created in full screen size, except that if running on the desktop the default size of the main window is set by the **set-env** script."

Thinking this is best mentioned after the  DALI_WINDOW_WIDTH=1920 DALI_WINDOW_HEIGHT=1080 my-app line as don't think most new users will know set-env well.

**Review by agnelovaz — CHANGES_REQUESTED** [review]

**Review by agnelovaz — APPROVED** [review]

**Pull request merged**